Documentation/project-docs/using-nightly-builds.md

```diff
@@ -0,0 +1,244 @@
+
+To setup the SDK, download the zip, extract it somewhere, and add the root folder to your path. Alternatively, always fully qualify the path to `dotnet` in the root of this folder for all instructions in this document.
+
+Note: The installer will put `dotnet` globally in your path, which you might not want for dogfooding daily toolsets.
+
+After setting up `dotnet` you can verify you are using the newer version  with `dotnet --info`. The version should be greater than 2.0.0-*.
+
+## Setup the project
+
+1. Create a new project
+    - Create a new folder for your app
+    - `cd` into it and run `dotnet new console`
+
+2. Restore packages so that you're ready to play:
+
+```
+$ dotnet restore
+```
+
+## Consume the new build
+
+Edit your `Program.cs` to consume the new APIs, for example:
+
+```csharp
+using System;
+using System.Net;
+
+class Program
+{
+    static void Main(string[] args)
+    {
+        WebUtility.HtmlDecode("&amp;", Console.Out);
+        Console.WriteLine();
+        Console.WriteLine("Hello World!");
+    }
+}
+```
+
+Run the bits:
+
+```
+$ dotnet run
+```
+
+Rinse and repeat!
+
+## Advanced Scenario - Using a nightly build of Microsoft.NETCore.App
+
+If you use the above instructions, your application will run against the same .NET Core 2.0 runtime that comes with the latest SDK. That works fine to get up and running quickly. However, there are times when you need to use a nightly build of Microsoft.NETCore.App which hasn't made its way into the SDK yet. To enable this, there are two options you can take.
+
+### Option 1: Framework-dependent
+
+This is the default case for applications - running against an installed .NET Core runtime.
+
+0. You still need to install the prerequisite .NET Core SDK from above.
+1. Also, install the specific .NET Core runtime you require:
+    - https://github.com/dotnet/core-setup#daily-builds
+    - Remember the version number you picked, you'll need it below.
+2. Modify your `.csproj` to reference the nightly build of Microsoft.NETCore.App.
+
+```xml
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <RuntimeFrameworkVersion>2.0.0-beta-xyz-00</RuntimeFrameworkVersion> <!-- pick nightly build -->
+  </PropertyGroup>
+```
+
+```
+$ dotnet restore
+$ dotnet run
+```
+
+### Option 2: Self-contained
+
+In this case, the .NET Core runtime will be published alongside your application.
+
+0. You still need to install the prerequisite .NET Core SDK from above.
+1. Modify your `.csproj` to reference the nightly build of Microsoft.NETCore.App *and*
+make it self-contained.
+
+```xml
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <RuntimeFrameworkVersion>2.0.0-beta-xyz-00</RuntimeFrameworkVersion> <!-- pick nightly build -->
+    <RuntimeIdentifier>win7-x64</RuntimeIdentifier> <!-- make self-contained -->
```

Comment:
Isn't the preferred approach on 2.0 to use "portable" RID, i.e. `win-x64`? Though I guess it doesn't matter much for Windows.

Reply:
Yes, this should say "win-x64".